images/scripts/debian.setup

```diff
@@ -28,7 +28,8 @@ libvirt-dbus \
 network-manager \
 packagekit \
 pcp \
-policykit-1 \
+polkitd \
+pkexec \
```

Comment:
Thanks! However, let's make this more precise:
```
# policykit-1 was split into multiple packages; keep old name for Debian 11 and Ubuntu
Recommends: sudo | pkexec | policykit-1
```
I.e.  Please drop the `polkitd` package. It's not a COCKPIT_DEPS, neither conceptually nor technically. Enough daemons (like udisks2) already pull it in.

The split package [exists in Debian stable](https://packages.debian.org/search?keywords=pkexec) already. However, [ubuntu-2204](https://launchpad.net/ubuntu/+source/policykit-1/0.105-26ubuntu1.3) does not yet have that.

So this needs a special case for ubuntu-2204, similar to libblockdev-mdraid2 before.

Reply:
Thanks for pointing it out, I forgot about ubuntu :)